### ISSUE TITLE
zsh completion file overwritten by fish completion file in last two 17.06-rc's (edge)

### ISSUE THREAD

**Issue by BretFisher** [docker/for-mac#1764]
### Expected behavior
In `/Applications/Docker.app/Contents/Resources/etc/` I should see `docker.zsh-completion` as a zsh completion file.

### Actual behavior
`docker.zsh-completion` is there but is actually now incorrectly named, as it contains fish shell completion. Also, both zsh and bash completion files have executable bit set (755).

### Information
`Docker for Mac: version: 17.06.0-ce-rc5-mac16 (4cdec4294a50b2233146b09469b49937dabdebdd)`

```
cd /Applications/Docker.app/Contents/Resources/etc/

-rw-r--r--   1 bret  admin  11955 Jun 20 09:49 docker-compose.bash-completion
-rw-r--r--   1 bret  admin  18951 Jun 20 09:49 docker-compose.zsh-completion
-rw-r--r--   1 bret  admin  10353 Jun 20 09:49 docker-machine.bash-completion
-rw-r--r--   1 bret  admin  13538 Jun 20 09:49 docker-machine.zsh-completion
-rwxr-xr-x   1 bret  admin  93190 Jun 20 09:49 docker.bash-completion
-rwxr-xr-x   1 bret  admin  36449 Jun 20 09:49 docker.zsh-completion

cat docker.zsh-completion
# docker.fish - docker completions for fish shell
#
# This file is generated by gen_docker_fish_completions.py from:
# https://github.com/barnybug/docker-fish-completion
```

### Steps to reproduce the behavior

- Install `17.06.0-ce-rc5` or `17.06.0-ce-rc4`

**Comment by thaJeztah** [comment]
Thanks for reporting! I can confirm that file somehow contains the `fish` completions, not the `zsh` completions (also, there does not seem to be a file for `fish`.

Wondering if there's a wildcard match being used somewhere in the release script 🤔

**Comment by thaJeztah** [comment]
Found an issue in one of the packaging script, and opened a pull request 👍

**Comment by gtardif** [comment]
This will be fixed in the final 17.06
